### ISSUE TITLE
Host version check on proxy server

### ISSUE THREAD

**Issue by asamuzaK** [asamuzaK/withExEditorHost#64]
Continue from [Host version: Incompatible · Issue #118 · asamuzaK/withExEditor](https://github.com/asamuzaK/withExEditor/issues/118)

**Comment by asamuzaK** [comment]
@inkarkat

When you get
```
node -e console.log(process.env)
```

Do you have HTTP_PROXY and/or HTTPS_PROXY environment variables in it?

**Comment by inkarkat** [comment]
Yes, I have, `http_proxy`, `https_proxy`, `ftp_proxy` and exceptions in `no_proxy`.

**Comment by asamuzaK** [comment]
2nd try build.
```
npm i -g withexeditorhost@dev
```

Does this work?

Also check the browser log.
Expected output:
```
hostVersion: { isLatest: true, latest: "5.1.4", result: 1 }
```
* `isLatest` is boolean (not null)
* `latest` is string (not null)
* `result` is number

**Comment by asamuzaK** [comment]
@inkarkat 
Can you check with the 2nd try build?

Thanks

**Comment by inkarkat** [comment]
Yes, it's working with `withexeditorhost@5.1.5-a.2`, too. The output still has `null` in there:

```
{…}
​hostVersion: {…}
​​latest: null
​​result: 1
​​<prototype>: Object { … }
​<prototype>: Object { … }
main.js:741:19
```

And I still had ☑ _Notify Host Error_ configured (from my attempt to get the add-on running), and got an error during browser startup (that didn't impact the add-on functionality, though):

> write PRTO 140029161695040:error:14094419:SSL routnes:ssl3_read_bytes:tlsv1 alert access denied:.../deps/openssl/openssl/ssl/record/rec_layer_s3.c:1536:SSL alert number 49

**Comment by asamuzaK** [comment]
t looks like a proxy server configuration issue or a module issue used by withExEditorHost.
I'm not sure I can do anything about it on the host side.

The error occurs in the process of checking to see if a new host is present, and since fallback is working, there is no problem with the host's behavior itself.
However, on the withExEditor (add-on) side, you will not be notified that there is a new host available.

**Comment by asamuzaK** [comment]
I will merge the current PR and fix this issue.
If we find something we can do on the host side, lets file a new issue then.

**Comment by asamuzaK** [comment]
Released v5.1.5

@inkarkat 
You are using dev build so update host like below.

```
npm i -g withexeditorhost@latest
```

**Comment by inkarkat** [comment]
Thank you very much! I really appreciate this!
